src/test/java/de/tum/in/www1/artemis/StudentExamIntegrationTest.java

```diff
@@ -1003,5 +1003,81 @@ else if (quizQuestion instanceof MultipleChoiceQuestion) {
                 request.putWithResponseBody("/api/exercises/" + exercise.getId() + "/submissions/exam", quizSubmission, QuizSubmission.class, HttpStatus.OK);
             }
         }
+        return studentExam;
+    }
+
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void testDeleteExamWithStudentExamsAfterConductionAndEvaluation() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968855)

Reply:
See f9e39e2dbe10ffa13483affe2de6359ac2244213